scripts/codespell_whitelist.txt

```diff
@@ -0,0 +1,3 @@
+iff
+nd
+assignend
```

Comment:
Is this (and `nd`) a real world?

Reply:
Yes. libsolidity/codegen/ABIFunctions.cpp and test/libsolidity/SolidityEndToEndTest.cpp .
The former is a template tag, the latter a specific test that has to contain "nd".

Reply:
So it's not case sensitive? Perhaps you can change the test so that it does not contain nd? Where exactly does it occur?

Reply:
It occurs here: https://github.com/ethereum/solidity/blob/develop/test/libsolidity/SolidityEndToEndTest.cpp#L7779
The test case mentions "a specific bug" and I decided not to touch it not knowing what the bug was.